proctor-common/src/main/java/com/indeed/proctor/common/model/ConsumableTestDefinition.java

```diff
@@ -206,6 +207,14 @@ public boolean getSilent() {
         return silent;
     }
 
+    public void setDynamic(final boolean dynamic) {
+        this.isDynamic = dynamic;
```

Comment:
Should we make this |= to prevent setting this to false if tested with multiple filters

Reply:
I think the anyMatch should prevent this but may just be a good safety feature

Reply:
No. setDynamic should always honor the value to set. Otherwise, it will make it very hard to debug. If multiple filters happens to set it to different values, we will look into how to address that, but this won't be the right place for address such issues